exporter/splunkhecexporter/factory.go

```diff
@@ -171,6 +171,7 @@ func createLogsExporter(
 		telemetrySettings: set.TelemetrySettings,
 		buildInfo:         set.BuildInfo,
 	}
+	c.heartbeater = newHeartbeater(cfg, c.pushLogData)
```

Comment:
you might want to also create one for the metric exporter?

Reply:
Right I should add them in traces and metric as well.